modules/base/src/main/scala/io/janstenpickle/trace4cats/base/context/Lift.scala

```diff
@@ -20,4 +20,11 @@ trait Lift[Low[_], F[_]] extends ContextRoot { self =>
 
 object Lift {
   def apply[Low[_], F[_]](implicit ev: Lift[Low, F]): Lift[Low, F] = ev
+
+  trait IdLift[F[_]] extends Lift[F, F] {
```

Comment:
What's the intention under defining `IdLift` separately as a trait? Why in the subnamespace?

The same question to `IdUnlift`.

Do we maybe need just type aliases for brevity?

I think `IdLift` name is misleading. It may be confused with `type IdLift[F[_]] = Lift[Id, F]`